bin/cylc

```diff
@@ -237,6 +237,7 @@ utility_commands['cycle-point'] = [
 utility_commands['scp-transfer'] = ['scp-transfer']
 utility_commands['suite-state'] = ['suite-state']
 utility_commands['ls-checkpoints'] = ['ls-checkpoints']
+utility_commands['show-times'] = ['show-times']
```

Comment:
FWIW my vote would be to rename to `cylc report-timings` and then modify into a `report` subcommand once we have something else ready to go.